src/components/misc/personViews/PersonViewTable.jsx

```diff
@@ -162,7 +165,7 @@ export default class PersonViewTable extends React.Component {
                                 const joinedOptions0 = latest0.selected.reduce(joinText, '');
                                 const joinedOptions1 = latest1.selected.reduce(joinText, '');
                                 x = joinedOptions0.localeCompare(joinedOptions1);
-                            }
+                            } 
```

Comment:
I'm not happy about little things like this, but I'm glad we're avoiding them in app.zetkin.org thanks to linting and auto-formatting. 😊